### DISCUSSION TITLE
Add a way to cite this repository

### DISCUSSION THREAD

**Discussion by IanViotti** [python-control/python-control#792] [Ideas]
I'm writing an academic paper for control systems and I used this library in my project. I know it's crucial to the development of open-source software to have proper citation and acknowledgment. Implementing this simple feature could be very beneficial to this community.

Cheers!

**Comment by murrayrm** [comment]
Thank you for taking the time to cite this work.  You can use the following:

S. Fuller, B. Greiner, J. Moore, R. Murray, R. van Paassen, R. Yorke. The Python Control Systems Library (python-control). 60th IEEE Conference on Decision and Contorl (CDC), pp 4875–4881, 2021.  Available online: https://python-control.org

It's also fine to just site the GitHub site: https://github.com/python-control/python-control

**Reply by IanViotti** [reply]
Thank you!
